### PR TITLE
Update README.md

### LINKED ISSUE #23: Update the Readme accordingly to new commits

### PR DESCRIPTION
**Pull request by ashwani99**
Fixes #23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,32 +16,46 @@ This assumes that you have `Go` installed and setup.
...
 
 ```bash
+$ make all
+$ dexer
+```
 
-make clean # for cleaning up
-make dev # for development purposes
-make logs # for getting the dummy directory ready
+Open http://localhost:8000 in your browser to open the user interface.
```
Comment (hellozee) [inline README.md]: Add a screenshot for the UI

Comment (hellozee) [reply README.md]: Use a service like https://imgbb.com/ for uploading the image and link it to the readme after it

File: README.md [hunk 2]
```diff
@@ -16,32 +16,46 @@ This assumes that you have `Go` installed and setup.
...
 
+The default configurations can be changed by editing the `config.json` file.
+```
+{
+    "RootDirectory": "logs",
+    "IndexFilename": "irclogs.bleve",
+    "Port": ":8000"
+}
 ```
```
Comment (hellozee) [inline README.md]: Add which entry is used for what

File: README.md [hunk 3]
```diff
@@ -16,32 +16,46 @@ This assumes that you have `Go` installed and setup.
...
+### Run locally using docker
+
+You can run the application using Docker in your local machine. It will use the `Dockerfile` instructions. Make sure you have [Docker](https://www.docker.com/) installed in your machine.
+
+Run the following commands to build and run the docker image.
+
+```bash
+$ make docker-build # builds docker image
+$ make docker-run # runs the image in new container
+```
```
Comment (hellozee) [inline README.md]: Add a section for development, write about how to generate a temporary folder with dummy data and test it

**Comment by hellozee** [comment]
since #26 is solved, please add the configuration entry to the documentation, @ashwani99

File: README.md [hunk 4]
```diff
@@ -16,32 +16,46 @@ This assumes that you have `Go` installed and setup.
...
+    "IndexFilename": "irclogs.bleve",
+    "Port": ":8000"
+}
 ```
 
-Once you did this you will notice that there is a `logs` directory created with different folder and each folder will have different files. Remember all these files are dummy files and they have some random junk.
+### API
 
-Then run `make build` this should start the server. Once everything is working fine install the postman plugin for your browser. And from that plugin you need to hit the endpoint as:
+Once everything is working fine install the postman plugin for your browser. And from that plugin you need to hit the endpoint as:
```
Comment (farhaanbukhsh) [inline README.md]: After having the UI we don't need postman to test either we move this somewhere down or remove it.

**Comment by farhaanbukhsh** [comment]
This looks good if we could elaborate  about config json that would be great too.

File: README.md (hunk 4 above)
Comment (ashwani99) [reply README.md]: @farhaanbukhsh @hellozee I think just describing the endpoint should be fine.

**Comment by farhaanbukhsh** [comment]
Hey @ashwani99 still working on it?

**Comment by farhaanbukhsh** [comment]
We got some really nice changes :) should someone else take this PR since you don't have time now? @ashwani99 @hellozee

**Pull request merged**